run_scripts/run_pipeline.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ $# -ne 10 ]]; then
+    echo "Usage: ./run_pipeline.sh"
+    echo "  <user> <coda-pull-credentials-path> <coda-push-credentials-path> <avf-bucket-credentials-path>"
+    echo "  <drive-service-account-credentials-url> <rapid-pro-tools-root> <coda-tools-root> <data-root>"
+    echo "  <drive-upload-dir> <data-backup-dir>"
+    echo "Runs the pipeline end-to-end (data fetch, coda fetch, output generation, Drive upload, Coda upload, data backup)"
+    exit
+fi
+
+USER=$1
+CODA_PULL_CREDENTIALS_PATH=$2
```

Comment:
I'm not sure it's worth separating pull and push credential at this point. The way we're interacting with Firebase means that if you have read permission to Coda, you also have write permissions

Reply:
The reason I did this was so I could pull the latest labelled data from the production instance, but still be able to push to the staging version during development.